tests/test_filtsmooth/filtsmooth_testcases.py

```diff
@@ -56,22 +55,17 @@ def car_tracking():
         proc_noise_cov_mat=measdiff,
     )
     initrv = Normal(mean, cov)
-    return dynmod, measmod, initrv, {"dt": delta_t, "tmax": 20}
 
-
-class CarTrackingDDTestCase(unittest.TestCase, NumpyAssertions):
-    """Car tracking: Ex.
-
-    4.3 in Bayesian Filtering and Smoothing
-    """
-
-    def setup_cartracking(self):
-        self.dynmod, self.measmod, self.initrv, info = car_tracking()
-        self.delta_t = info["dt"]
-        self.tms = np.arange(0, 20, self.delta_t)
-        self.states, self.obs = pnss.generate_samples(
-            self.dynmod, self.measmod, self.initrv, self.tms
-        )
```

Comment:
@pnkraemer Correct me if I am wrong, but this (and, analogously, the `OrnsteinUhlenbeckCDTestCase`) is not used, right?